### PR TITLE
Basic tests

### PR DESCRIPTION
**Pull request by savage13**
Added 3 tests:
- Basic Orb Duping
  - Taken from the initial data on statup
- Getting 999 Apples
- Getting 999 Apples and many Orbs

Note: I needed to use `Sync GameData` to make sure the basic orb duping test worked as result had a difference between the `Game Data` and `Visible Inventory`.  Is there a set the `Game Data` differently than the `Visible Inventory`? Or am I thinking about this incorrectly?

### CODE REVIEW

File: src/__tests__/apples_999.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Author: Swiffy22
```
Comment (iTNTPiston) [inline src/__tests__/apples_999.e2e.ts:1]: change this to Savage13

File: src/__tests__/apples_orbs.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Author: Swiffy22
```
Comment (iTNTPiston) [inline src/__tests__/apples_orbs.e2e.ts:1]: change this to Savage13

File: src/__tests__/dup_orbs.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Author: Swiffy22
```
Comment (iTNTPiston) [inline src/__tests__/dup_orbs.e2e.ts:1]: change this to Savage13

**Review by iTNTPiston — CHANGES_REQUESTED** [review]
We don't have a command that initializes gamedata to be different yet. I think im going to make that now. You can change the tests once that's there

File: src/__tests__/apples_999.e2e.ts (hunk 1 above)
Comment (iTNTPiston) [reply src/__tests__/apples_999.e2e.ts:1]: or savage13 idk which one you prefer

File: src/__tests__/apples_orbs.e2e.ts (hunk 2 above)
Comment (savage13) [reply src/__tests__/apples_orbs.e2e.ts:1]: Done

File: src/__tests__/apples_999.e2e.ts (hunk 1 above)
Comment (savage13) [reply src/__tests__/apples_999.e2e.ts:1]: Done

File: src/__tests__/dup_orbs.e2e.ts (hunk 3 above)
Comment (savage13) [reply src/__tests__/dup_orbs.e2e.ts:1]: Done

**Comment by iTNTPiston** [comment]
I'm a bit confused about the commit history, looks like you have some commits duplicated? Can you try rebase on latest main?

**Pull request closed without merging**